README.md

```diff
@@ -47,48 +47,38 @@ This simple test HTML page can also be hosted on your environment to showcase ht
 ```
 
 ## Developer Setup
-This section provides step by step guide on setting up this repo and create simple local hosting.
-You need [npm](https://www.npmjs.org/) installed on your computer.
-From the root project directory run these commands from the command line:
+1. Install [brew](http://brew.sh): 
```

Comment:
Brew is an OSX specific dependency manager. This ties our setup instructions specifically to OSX, which I disagree with. We shouldn't be giving instructions that only work on one of three major OS options.